src/index.html

```diff
@@ -2,16 +2,146 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="field-set">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+          </label>
+          <input type="text" name="surname"
+          id="surname" autocomplete="disabled">
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+          </label>
+          <input type="text" name="name"
+          id="name" autocomplete="disabled">
+        </div>
+        <div class="form-field">
+          <label for="age">
+            How old are You?
+          </label>
+          <input type="number" name="age"
+          id="age" value="12"
+          max="100" min="0">
+        </div>
+        <div class="form-field">
+          <label for="date-of-birth">
+            Full date of birth:
+          </label>
+          <input type="date" name="Full date of birth"
+          id="date-of-birth">
+        </div>
+        <div class="form-field">
+          <label for="terms">
+            I accept the term of the agreement
+          </label>
+          <input type="checkbox" name="terms"
+          id="terms">
+        </div>
+      </fieldset>
+      <fieldset class="field-set">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+          </label>
+          <input type="email" name="email"
+          id="email" placeholder="email@example.com"
+          required>
```

Comment:
```suggestion
          <input
            type="email"
            name="email"
            id="email"
            placeholder="email@example.com"
            required
          >
```
Форматируй теги, в которых >= 3 атрибутов таким образом